backend/src/main/java/com/bakdata/conquery/util/search/TrieSearch.java

```diff
@@ -183,8 +183,8 @@ Stream<String> suffixes(String word) {
 		return Stream.concat(
 				// We append a special character here marking original words as we want to favor them in weighing.
 				Stream.of(word + WHOLE_WORD_MARKER),
-				IntStream.range(1, Math.max(1, word.length() - suffixCutoff))
-						 .mapToObj(word::substring)
+				IntStream.range(1, Math.max(1, word.length() - ngramLength))
+						 .mapToObj(start -> word.substring(start, start + ngramLength))
```

Comment:
In den Tests sieht es so aus, als würden die ngrams nicht wieder concatiniert werden.

Reply:
weiß nicht was das bedeutet, aber die umstellung auf ngrams ist nerviger als ich dachte, muss noch rumspielen falls ich das umsetzen will. Speicher sparen tut es bei sehr großen Strings enorm viel